Docs.md

```diff
@@ -155,6 +155,9 @@ Other useful commands:
     ./third_party/depot_tools/gn desc out/debug/ :deno
     ./third_party/depot_tools/gn help
 
+    # Update third_party modules
+    ```$ git submodule update```
```

Comment:
For consistency no code ticks or $ required. It's already a code block by indentation.

Reply:
done